<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Trend</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLabel" name="gustLabel">
     <property name="text">
      <string>Gust</string>
     </property>
     <property name="buddy">
      <cstring>gust</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="weatherWithIntensityLabel">
     <property name="text">
      <string>Weather</string>
     </property>
     <property name="buddy">
      <cstring>weatherWithIntensity</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="weatherLabel">
     <property name="text">
      <string>Weather</string>
     </property>
     <property name="buddy">
      <cstring>weather</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="periodLabel">
     <property name="text">
      <string>Interval</string>
     </property>
     <property name="buddy">
      <cstring>period</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="windLabel">
     <property name="text">
      <string>Wind</string>
     </property>
     <property name="buddy">
      <cstring>wind</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="visLabel">
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="buddy">
      <cstring>vis</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="cloud1Label">
     <property name="text">
      <string>Cloud</string>
     </property>
     <property name="buddy">
      <cstring>cloud1</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="cloud2Label">
     <property name="text">
      <string>Cloud</string>
     </property>
     <property name="buddy">
      <cstring>cloud2</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="vis"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="period">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="wind"/>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="cbLabel">
     <property name="text">
      <string>Cumulonimbus</string>
     </property>
     <property name="buddy">
      <cstring>cb</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="cloud1"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="nosig">
     <property name="text">
      <string notr="true">NOSIG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="cloud3Label">
     <property name="text">
      <string>Cloud</string>
     </property>
     <property name="buddy">
      <cstring>cloud3</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLineEdit" name="cloud2"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="gust"/>
   </item>
   <item row="4" column="6">
    <widget class="QCheckBox" name="nsc">
     <property name="text">
      <string notr="true">NSC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLineEdit" name="cloud3"/>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="weather"/>
   </item>
   <item row="3" column="9">
    <widget class="QLineEdit" name="cb"/>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="weatherWithIntensity"/>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="cavok">
     <property name="text">
      <string notr="true">CAVOK</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="typeGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QRadioButton" name="becmg">
        <property name="text">
         <string notr="true">BECMG</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="tempo">
        <property name="text">
         <string notr="true">TEMPO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QGroupBox" name="prefixGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QCheckBox" name="at">
        <property name="text">
         <string notr="true">AT</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fm">
        <property name="text">
         <string notr="true">FM</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="tl">
        <property name="text">
         <string notr="true">TL</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="10">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>becmg</tabstop>
  <tabstop>tempo</tabstop>
  <tabstop>fm</tabstop>
  <tabstop>tl</tabstop>
  <tabstop>at</tabstop>
  <tabstop>period</tabstop>
  <tabstop>wind</tabstop>
  <tabstop>gust</tabstop>
  <tabstop>vis</tabstop>
  <tabstop>cavok</tabstop>
  <tabstop>weatherWithIntensity</tabstop>
  <tabstop>weather</tabstop>
  <tabstop>cloud1</tabstop>
  <tabstop>cloud2</tabstop>
  <tabstop>cloud3</tabstop>
  <tabstop>cb</tabstop>
  <tabstop>nsc</tabstop>
  <tabstop>nosig</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
